packages/xchain-polkadot/src/util.ts

```diff
@@ -1,4 +1,15 @@
+import { BaseAmount, AssetAmount, baseAmount } from '@xchainjs/xchain-util/lib'
+
 /**
- * Type guard for runtime checks of `Fee`
+ * Check Subscan API response
  */
 export const isSuccess = (response: { code: number }): boolean => !response.code
+
+/**
+ * convert asset to base values (decimal = 10)
+ */
+export const assetToBase10 = (v: AssetAmount): BaseAmount => {
+  const value = v.amount().multipliedBy(Math.pow(10, v.decimal)).integerValue()
+
+  return baseAmount(value, 10)
+}
```

Comment:
You don't need this helper, if you use [assetToBase](https://github.com/xchainjs/xchainjs-lib/blob/6831adb99d43292bc185d3194f3ad40fddf0e191/packages/xchain-util/src/asset.ts#L62) of `xchain-util`